py/desispec/maskbits.py

```diff
@@ -68,6 +69,13 @@
     - [BADAMPB,        22, "Issues in the amplifier readouts of camera B make this unusable"]
     - [BADAMPR,        23, "Issues in the amplifier readouts of camera R make this unusable"]
     - [BADAMPZ,        24, "Issues in the amplifier readouts of camera Z make this unusable"]
+#- Following bits are reserved for spectro QA
+    - [BADPETALPOS,    25, "Too many fibers with bad positioning (bits 1,2,3,8,9) in petal"]
```

Comment:
See earlier comment about which bits should be included in BADPETALPOS; update here as well if we change the config definition (or cite config here instead of replicating which bits are bad)